app/helpers/kuroko2/job_definitions_helper.rb

```diff
@@ -34,5 +34,54 @@ def star_link_for(definition)
         )
       end
     end
+
+    def format_kuroko_script(script_text)
+      script_text.each_line.map { |line|
+        matched = Kuroko2::Workflow::ScriptParser::LINE_REGEXP.match(line.chomp)
+        case matched && matched[:type]
+        when 'wait'
+          format_wait_task(line, matched)
+        when 'sub_process'
+          format_sub_process_task(line, matched)
+        else
+          ERB::Util.html_escape(line)
+        end
+      }.join('').html_safe
+    end
+
+    private
+
+    def format_wait_task(line, matched)
+      definition_names = []
+      formatted_line = line.gsub(Kuroko2::Workflow::Task::Wait::OPTION_REGEXP) { |option|
+        definition = Kuroko2::JobDefinition.find_by(id: $1.to_i)
+        if definition.present?
+          definition_names << definition.name
+          link_to(option, job_definition_path(definition.id))
+        else
+          option
```

Comment:
Is `ERB::Util.html_escape` necessary for `option`? `format_sub_process_task` looks to escape similar case.